homeassistant/components/environment_canada/sensor.py

```diff
@@ -91,38 +56,47 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     from env_canada import ECData
 
     if config.get(CONF_STATION):
-        ec_data = ECData(station_id=config[CONF_STATION])
+        ec_data = ECData(station_id=config[CONF_STATION],
+                         language=config.get(CONF_LANGUAGE))
     elif config.get(CONF_LATITUDE) and config.get(CONF_LONGITUDE):
         ec_data = ECData(coordinates=(config[CONF_LATITUDE],
-                                      config[CONF_LONGITUDE]))
+                                      config[CONF_LONGITUDE]),
+                         language=config.get(CONF_LANGUAGE))
     else:
         ec_data = ECData(coordinates=(hass.config.latitude,
-                                      hass.config.longitude))
+                                      hass.config.longitude),
+                         language=config.get(CONF_LANGUAGE))
 
-    add_devices([ECSensor(sensor_type, ec_data, config.get(CONF_NAME))
-                 for sensor_type in config[CONF_MONITORED_CONDITIONS]],
+    sensor_list = list(ec_data.conditions.keys()) + list(ec_data.alerts.keys())
+    add_devices([ECSensor(sensor_type,
```

Comment:
Please rename `add_devices` to `add_entities`.

Reply:
Done